src/astro/files/__init__.py

```diff
@@ -0,0 +1,92 @@
+import os
+import pathlib
+from typing import List, Optional
+
+import pandas as pd
+import smart_open
+
+from astro.constants import LOAD_DATAFRAME_BYTES_LIMIT, FileType
+from astro.files.locations import location_factory
+from astro.files.type import type_factory
+from astro.utils.file import get_size
+
+
+class File:
```

Comment:
Is there a reason that this class exists in the __init__ file? Are we planning for this to part of the public API?

Reply:
Moved it in base.py file